src/detail-panels/account-header/account-header.jsx

```diff
@@ -31,9 +31,9 @@ export function AccountHeader({
   console.log(resolved.data)
   const handleHistoryQuery = useHandleHistory(resolved.data?.shortDID);
   const handleHistory = handleHistoryQuery.data?.handle_history;
-
+ 
   const placementquery = usePlacement(resolved.data?.shortDID) ;
-  const placement = placementquery.data?.placement.toLocaleString() ?? ""; 
+  const placement = placementquery.data?.placement ? placementquery.data?.placement.toLocaleString() : ""; 
```

Comment:
```suggestion
  const placement = placementquery.data?.placement?.toLocaleString() ?? ""; 
```

This will stay easier to read with just an extra optional accessor instead of a ternary